openapi/src/main/scala/endpoints/delegate/DocumentedEndpoints.scala

```diff
@@ -0,0 +1,30 @@
+package endpoints
+package delegate
+
+import endpoints.algebra.MuxRequest
+
+/**
+  * Interpreter for [[algebra.DocumentedEndpoints]] that ignores information
+  * related to documentation and delegates to another [[algebra.Endpoints]]
+  * interpreter.
+  */
+trait DocumentedEndpoints
+  extends algebra.DocumentedEndpoints
+    with DocumentedRequests
+    with DocumentedResponses {
+
+  val delegate: algebra.Endpoints
```

Comment:
Users can turn a `DocumentedXxx` into an `Xxx` using delegation. This makes it possible to apply the usual interpreters (e.g. `play.server.Endpoints`) to a `DocumentedEndpoints`.